sw/device/lib/dif/dif_pinmux.h

```diff
@@ -360,15 +360,16 @@ dif_result_t dif_pinmux_output_select(const dif_pinmux_t *pinmux,
  * @param pinmux A Pin Multiplexer handle.
  * @param pad Pad index to write the attributes for.
  * @param type Pad type (MIO or DIO).
- * @param attrs_in Attributes to write. A set bit writes an attribute, a
- *                 clear bit leaves it in its existing state.
+ * @param attrs_mask Mask of attributes to write.
+ * @param attrs_in Attribute values to write.
  * @param attrs_out[out] The actual attributes written and accepted by the
  *                       hardware.
  * @return The result of the operation.
  */
 dif_result_t dif_pinmux_pad_write_attrs(const dif_pinmux_t *pinmux,
                                         dif_pinmux_index_t pad,
                                         dif_pinmux_pad_kind_t type,
+                                        dif_pinmux_pad_attr_mask_t attrs_mask,
```

Comment:
Not sure why I did that. We can do read-modify-write with `dif_pinmux_pad_get_attrs`. Removed.